Oracle_CQRS/pom.xml

```diff
@@ -0,0 +1,129 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>org.springframework.boot</groupId>
+        <artifactId>spring-boot-starter-parent</artifactId>
+        <version>3.4.5</version>
+        <relativePath/> <!-- lookup parent from repository -->
+    </parent>
+    <groupId>org.example</groupId>
+    <artifactId>Oracle_CQRS</artifactId>
+    <version>0.0.1-SNAPSHOT</version>
+    <name>Oracle_CQRS</name>
+    <description>Oracle_CQRS</description>
+    <url/>
```

Comment:
Either fill these properties or remove them, for samples you can use the [UPL](https://oss.oracle.com/licenses/upl/)